packages/matter-node.js/src/Device.ts

```diff
@@ -43,22 +43,29 @@ import { AdminCommissioningHandler } from "./matter/cluster/server/AdminCommissi
 import { NetworkCommissioningHandler } from "./matter/cluster/server/NetworkCommissioningServer";
 import { AttestationCertificateManager } from "./matter/certificate/AttestationCertificateManager";
 import { CertificationDeclarationManager } from "./matter/certificate/CertificationDeclarationManager";
+import { StorageNode } from "./persistence/StorageNode";
```

Comment:
Have you considered using the `node-localstorage` package as `matternode` did?  It is a ready-to-go package, and every field is a simple text file in storage (so nicely independent from async issues, and easy to view / patch by the user out-of-band).

https://github.com/project-chip/matternode/blob/main/src/platform/PersistantStore.js#L35

It would be good to make sure a platform abstraction is part of the design or at least there is a plan for how such a PAL could be added later.

Reply:
The approach from node-matter so far was "as less dependencies as possible", thats why no other packages were considered. It was decided to do a very lightweight nodejs storage as "defaule basic implementation". But in fact as soon as the Storage interface is implemented the dev later on can switch out the real Storage implementation (like Storagenode.ts now) by any other implementation.

Reply:
Yeah my concern is that there is quite a bit of new code to support this JSON format, all of which needs to have tests and docs, and will only potentially apply to the node impl, and not web or other platforms. `node-localstorage` has really easy to use and understand footprint on disk.  I'll have to play with a local copy of this PR to see what the storage files look like in practice.

Reply:
In fact it is "a big json". Does it need to be readable?

Reply:
(But we can also use a lib, I have no issue with this)

Reply:
replaces by node-localstorage